README.md

```diff
@@ -1,25 +1,104 @@
 # Scripts to manage GitHub Classroom
 
-This folder contains scripts to collect assignments and push feedback to student repositories in a given GitHub organization. There are no repo creation scripts here, since we use GitHub classroom for this part. Look at [https://github.com/snadi/UsefulCourseScripts](https://github.com/snadi/UsefulCourseScripts) if you are interested in that.
+This folder contains scripts to collect assignments or labs and push feedback to student repositories in a given GitHub organization. There are no repository creation scripts here, since we use GitHub classroom for this part. 
 
-# Prerequisites
-
-The following python libraries are being used:
+## Dependencies
 
+- Python 3
 - [github3](https://github.com/sigmavirus24/github3.py)
 
-Install packages with: `pip3 install -r requirements.txt`
+````shell
+# Quick installation steps
+pip3 install -r requirements.txt
+````
+
+These scripts have been tested with Python 3.9.1(latest).
+
+
+
+## Directory structure
+
+```
+.
+├── ccid_list.txt               # Contains ccids you need to grade
+├── collect_repos.py            # Collect repos
+├── config                      # Configration file for collect_repos.py
+├── push_feedback.sh            # Push feedback to all repos
+├── README.md                   # This file
+├── requirements.txt            # For dependencies installation
+├── student_list.csv            # A mapping from ccids to github usernames
+├── token_file.txt              # Your Github access token(You need to create this file)
+├── update_repos.sh             # Update all repos
+│
+├── assignment1                 # A subdirectory named with an assignment or lab name 						
+│   │                           # It's automatically created after running collect_repos.py
+│   ├── repo1 
+│   │   ├── feedback.txt        # Feedback you made
+│   │   └── ...
+│   ├── repo2                   # Student's repository	
+│   └── ...                
+└── ...                         # Other subdirectories (e.g., labs, assignment2, etc.)
+```
+
+
+
+## Instructions
+
+To use these scripts, please follow the following steps:
+
+1. **Generate a personal access token**	
+
+   - Go to https://github.com/settings/tokens, click `Generate new token`, select all scopes and click `Generate token`.
+
+   - Remember to copy the token and paste it to a file created by yourself named `token_file.txt`.
+
+2. **Configure**
+
+   - Open `config`, make your own configuration. Typically, you need to change `username`, `prefix` and `deadline`.
+   - `prefix` is the name of lab or assignment you are grading (e.g., assignment1).
+   - If you want to collect all repos, provide no value to key `list`.
```

Comment:
Otherwise, see ccid instructions below? It doesn't say, what the value of list would be if you don't want to collect all repos